.github/dependabot.yml

```diff
@@ -2,9 +2,9 @@
 version: 2
 
 updates:
-  # Check for updates to GitHub Actions.
-  - package-ecosystem: "github-actions"
-    directory: "/"
-    schedule:
-      interval: "daily"
-    open-pull-requests-limit: 10
+    # Check for updates to GitHub Actions.
+    - package-ecosystem: 'github-actions'
+      directory: '/'
+      schedule:
+          interval: 'daily'
+      open-pull-requests-limit: 10
```

Comment:
I believe this change was not intended, but was auto-formatted by lint-staged or something.
But I think the indentation and quoting is correct after the change.

Reply:
It's done that to me too. It looks good. If you want to move the change to a separate PR, I'll be happy to merge it.